shared/ava.config.mjs

```diff
@@ -1,4 +1,5 @@
 export default {
+  workerThreads: false,
```

Comment:
@lukastaegert this change broke tests throughout the repo. symlinks that pointed to the old file are now broken. this didn't need to be changed either, as rollup doesn't use it directly